dali/test/python/test_dali_tf_es_pipelines.py

```diff
@@ -0,0 +1,185 @@
+# Copyright (c) 2021, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Maybe this file could have been named like utils or something?

Reply:
I keep the test pipelines here, hence the name.